AcadeMeData/test_msgtags.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from AcadeMeData.models import MessageTags, Messages, User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com", 'type': "S",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.mark.django_db
+class TestMessageTagsModel:
+    def test_get_msg_tag(self, user_example, id=1, msgID=None):
```

Comment:
You should call fixtures here 
the test should call the create_message fixture that calls create_user  
Once you have a tag, you should search it and receive it, where is the test? i mean, if there is a tag_id you search, you have to verify the same user created, isnt it?

I mean if you have message tag